### DISCUSSION TITLE
High Availability with a small node

### DISCUSSION THREAD

**Discussion by sushiMix** [nats-io/nats-server#4982] [General]
Hello, 
we would like to deploy nats (with jet-stream in high availability).
As it based on quorum we can't deploy with two nodes https://github.com/nats-io/nats-server/discussions/2869 and in our case we can have 3 nodes but the third node is very small. Is there a tuning to allow to deploy a "voting only" node for nats jetstream ?

**Comment by Jarema** [comment]
Hey!

That is not possible, as each replica needs to store all the messages too.

This might be interesting for you: https://www.youtube.com/watch?v=uWhUJvGnwLY
